fog/ledger/server/src/router_service.rs

```diff
@@ -80,3 +84,94 @@ where
         });
     }
 }
+
+/// Used for the implementation of FogKeyImageApi::check_key_images(),
+/// the legacy unary key-image API, for LedgerRouterService.
+async fn unary_check_key_image_inner<E>(
```

Comment:
Recommend renaming to `..._impl`. We don't usually use `inner` to denote a function that is exclusively used by another function.

Reply:
Sure. (@NotGyro , for future reference).